### DISCUSSION TITLE
TeamTalk v5.12.0 beta not available for Ubuntu 18

### DISCUSSION THREAD

**Discussion by Nardol** [BearWare/TeamTalk5#1603] [General]
Hi @bear101,

Can you make Teamtalk 5.12.0 beta available for Ubuntu 18 please?
The version for Ubuntu 20.04 is unfortunately not compatible with Debian at all.

**Comment by bear101** [comment]
Can't you use the output from GitHub Actions?

**Reply by Nardol** [reply]
If I remember correctly, Github actions don't provide WebRTC and without it, sound experience is really not the same...

**Comment by lbk2907** [comment]
Excuse me, may I know how to know the WebRTC is fully supported? Thanks.

**Reply by Nardol** [reply]
Compare filters performances between the beta version on bearware.dk and the GH action output.
Or read the code of Github actions and see WebRTC is disabled: `make CMAKE_EXTRA="-DFEATURE_WEBRTC=OFF -DCMAKE_INSTALL_PREFIX=${{runner.workspace}}/install" -C Build ubuntu18`
More precisely: `-DFEATURE_WEBRTC=OFF`

**Comment by lbk2907** [comment]
Thanks.
